tests/commands/vmmap.py

```diff
@@ -23,4 +23,4 @@ def test_cmd_vmmap(self):
         self.assertGreater(len(res.splitlines()), 1)
 
         res = gdb.execute("vmmap $pc", to_string=True)
-        self.assertEqual(len(res.splitlines()), 2)
+        self.assertEqual(len(res.splitlines()), 8)
```

Comment:
Need some tests for `-n` and `-a` and then we're good

Reply:
Oh lol I forgot to push this x)